samples/Chat/src/app/ConfigurationScreen.tsx

```diff
@@ -129,6 +174,16 @@ export default (props: ConfigurationScreenProps): JSX.Element => {
           return;
         }
         setConfigurationScreenState(CONFIGURATIONSCREEN_SHOWING_JOIN_CHAT);
+
+        // Join with URL Parameters 
+        const token = getExistingTokenFromURL();
```

Comment:
If we are invoking these here to check if we have all the params we need could we not just pass them into the `joinChatThreadWithExistingUser` callback?